bindings/python/eblob_python.cpp

```diff
@@ -52,6 +53,29 @@ static void eblob_extract_id(const struct eblob_id &e, struct eblob_key &id)
 	eblob_extract_arr(e.id, id.id, &len);
 }
 
+/**
+ * __eblob_read_ll() - interruption-safe wrapper for pread(2)
+ */
+static int __eblob_read_ll(int fd, char *data, size_t size, off_t offset)
```

Comment:
> How will it behave when I press Ctrl-C while python binding reads data from fd?

As far as I know ignoring EINTR after syscalls will not affect unix process signal handling. So if SIGINT happens, then process will do some default action (process interrupt) or will do something else (e.g. ignoring signal or something user-defined).